releasenotes/notes/backend-estimator-v2-variance-905c953415ad0e29.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Changes the way in which the `BackendEstimatorV2` class calculates the `std` to ensure that
```

FYI: Reno uses sphinx format not markdown.

```suggestion
    Changes the way in which the :class:`.BackendEstimatorV2` class calculates the ``std`` to ensure that
```

---

Thank you for this!